samples/GraphQL.DataLoader.Example/Program.cs

```diff
@@ -0,0 +1,43 @@
+using DataLoaderGql;
+using DataLoaderGql.GraphQl;
+using GraphQL;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddScoped<SalespeopleByNameDataLoader>().AddScoped<CarsBySalespersonDataLoader>();
+// Add services to the container.
+// Learn more about configuring OpenAPI at https://aka.ms/aspnet/openapi
+builder.Services.AddDbContext<DealershipDbContext>();
+
+builder.Services.AddGraphQL(builder =>
+{
+    builder
+        .AddSystemTextJson()
+        .AddGraphTypes(typeof(CarsGraphType).Assembly)
+        .AddSchema<DealershipSchema>();
+});
```

Comment:
This should be formatted like all of our examples, as follows:
```suggestion
builder.Services.AddGraphQL(b => b
    .AddSystemTextJson()
    .AddGraphTypes(typeof(CarsGraphType).Assembly)
    .AddSchema<DealershipSchema>()
);
```